kuma/static/layout-tests/iex-unused-css.html

```diff
@@ -552,8 +552,8 @@ <h1>Array.prototype.forEach()</h1>
+<style>.interactive{color:#333;margin:0;padding:0;height:100%;font:16px/1.5 sans-serif}.interactive header h4:before{display:inline-block;content:'';background:transparent url(../../../img/interactive-examples/dino.svg) 0 0 no-repeat;margin-right:.6em;width:45px;height:45px;vertical-align:middle}.interactive header h4{display:inline-block;margin:0;font-size:.9rem;font-weight:700}.interactive .user-message{display:none;position:absolute;top:30px;left:50%;background-color:#333;color:#fff;padding:.5em;border-radius:4px;font-size:14px;opacity:1;transition:opacity .5s;z-index:4}.fade-in{animation:fadein .5s}.fade-out{animation:fadeout .5s}.hide{opacity:0}.show{display:block}.visually-hidden{position:absolute!important;height:1px;width:1px;margin:-1px;overflow:hidden;clip:rect(0 0 0 0);padding:0;border:0}@media screen and (min-width:47.9375em){.interactive header h4{font-size:1.1rem}}@media screen and (min-width:63.9375em){.interactive header h4{font-size:1.4rem}}@keyframes fadein{from{opacity:0}to{opacity:1}}@keyframes fadeout{from{opacity:1}to{opacity:0}}.editor,.output{font-size:14px}.editor{position:relative;margin-bottom:1em}.output{position:relative;display:inline-block;box-sizing:border-box;background-color:#fff;margin:0;padding:.8em;border:1px solid #eaf2f4;box-shadow:2px 2px 5px -2px rgba(0,0,0,.1);width:91%;height:120px;font-family:courier}.output code{width:100%;height:100px;white-space:pre;vertical-align:middle;word-break:break-word;overflow-y:auto;background:0 0}.buttons-container{display:inline-block;width:100px;vertical-align:top}.interactive .button{display:block;box-sizing:border-box;background-color:#fff;color:#333;padding:.5em;border:2px solid #333;width:100px;font-size:14px;line-height:1;cursor:pointer}.interactive .button:first-child{margin-bottom:.5em}.interactive .button:hover{background-color:#333;color:#fff}.js .editor{min-height:280px}.CodeMirror{border:1px solid #3f87a6;border-left:5px solid #3f87a6;height:280px}.js .editor.taller,.js .editor.taller .CodeMirror{height:490px}.js .editor.shorter,.js .editor.shorter .CodeMirror{height:200px;min-height:200px}@supports (display:flex){.buttons-container{align-self:flex-start;width:auto}.output-container{display:flex;align-items:center}.run{margin-right:10px}.output{display:flex;flex:10 150px;align-items:center}}</style>
```

Comment:
@hobinjk Just to confirm, the CSS here had been cleaned up to not include any unused CSS? If not, that is not a problem, we should then just change the filename to `iex-inlined.html` or something along those lines.

Reply:
That specific CSS has not. However, the rest of the CSS on the page has been cleaned up. I went for iex-unused-css since it's based on the unused-css page.